Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/saml/extraction/SamlTokenCreator.java

```diff
@@ -169,6 +169,21 @@
                         assertion.getSamlIssuer().getIssuerFormat());
                 }
             }
+            if (assertion.getSamlConditions() != null) {
+                if (assertion.getSamlConditions().getNotBefore() != null) {
+                    if (NullChecker.isNotNullish(assertion.getSamlConditions().getNotBefore())) {
```

Comment:
Suggestion: Remove duplicate check for not null in assertion.getSamlConditions().getNotBefore().